core/src/Model.cpp

```diff
@@ -119,11 +119,11 @@ void Model::configure()
 #ifdef USE_MPI
     std::string partitionFile
         = Configured::getConfiguration(keyMap.at(PARTITIONFILE_KEY), std::string("partition.nc"));
+    m_etadata.getPartitionMetadata(partitionFile);
```

Comment:
it's really not a big issue, but I am just curious. Why is the metadata called `m_etadata`?

Reply:
This is question to @timspainNERSC.

Reply:
<_< >_> 
No good reason.
Some classes, especially from early on in development, have a `m_` prefix for member variables. It slightly amused me to not repeat the 'm' in `m_metadata`.

Reply:
fair enough. Thanks for the answer.